netlify.toml

```diff
@@ -0,0 +1,15 @@
+[build]
+command = """
+    scripts/build-glean-metadata.py
+    npm run build
+    if [ "$STORYBOOK" ]; then
```

Comment:
For `glean-dictionary-dev` (only), I've set this variable to `1` in the GUI. This seems like the most straightforward way of getting the right behaviour for our use case (multiple netlify "apps" for the same repo)